prime-dotnet-webapi/Configuration/Database/Seeded Tables/StatusConfiguration.cs

```diff
@@ -14,7 +14,8 @@ public override IEnumerable<Status> SeedData
                     new Status { Code = 2, Name = "Under Review" },
                     new Status { Code = 3, Name = "Requires TOA" },
                     new Status { Code = 4, Name = "Locked"       },
-                    new Status { Code = 5, Name = "Declined"     }
+                    new Status { Code = 5, Name = "Declined"     },
+                    new Status { Code = 6, Name = "Disabled"     }
```

Comment:
How is "Disabled" `Status` used?

Reply:
It is used in column AuthorizedUser.Status